sites/ortec.js

```diff
@@ -1,72 +1,45 @@
-  Germany: "https://ortec.com/api/pages/en/careers/our-locations/germany",
-  Poland: "https://ortec.com/api/pages/en/careers/our-locations/poland",
-  "United Kingdom":
-    "https://ortec.com/api/pages/en/careers/our-locations/united-kingdom",
-};
-
-const generateJob = (job_title, job_link, country, city) => ({
-  job_title,
-  job_link,
-  country,
-  city,
-});
-
-const getJobs = async () => {
-  const jobs = [];
-  for (const [key, value] of Object.entries(urls)) {
-    const scraper = new Scraper(value);
-    const httpsAgent = new https.Agent({ rejectUnauthorized: false });
-    scraper.config.httpsAgent = httpsAgent;
-    const json = await scraper.get_soup("JSON");
-    const items = json.sections[4].jobs;
-
-    if (items) {
-      items.forEach((job) => {
-        const job_obj = generateJob(job.title, job.jobURL, key, job.subtitle);
-        jobs.push(job_obj);
-      });
-    }
+"use strict";
+const scraper = require("../peviitor_scraper.js");
+const company = { company: "Ortec" };
+let finalJobs = [];
+const apiKey = process.env.KNOX
+const url = 'https://ortec.com/api/career'
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+fetch(url, {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36'
   }
-  return jobs;
-};
-
-const getParams = () => {
-  const company = "Ortec";
-  const logo =
-    "https://media.academictransfer.com/LT8OEP2nAexUPaM9-WfgcP488FM=/fit-in/490x162/filters:upscale():fill(white)/logos/ortec-en-wide.jpg";
-  const apikey = process.env.APIKEY;
-  const params = {
-    company,
-    logo,
-    apikey,
-  };
-  return params;
-};
+})
+  .then(response => response.json())
+  .then(responseData => {
+
+    const filteredArray = responseData
+      .filter(obj => obj.country === 'Romania')
+      .map(({ title, jobId }) => ({ title, jobId }));
+
+    filteredArray.forEach(obj => {
+      const jobTitle = obj.title;
+      const jobid = obj.jobId;
+      finalJobs.push({
+        job_title: jobTitle,
+        job_link: "https://ortec.com/en/careers/find-jobs/career/jobs?id="+ jobid,
+        company: company.company,
+        city: 'Bucuresti',
+        country: "Romania",
+      });
+    });
+  }).then(() => {
+    console.log(JSON.stringify(finalJobs, null, 2));
 
-const run = async () => {
-  const jobs = await getJobs();
-  const params = getParams();
-  postApiPeViitor(jobs, params);
-};
+    scraper.postApiPeViitor(finalJobs, company, apiKey);
 
-if (require.main === module) {
-  run();
-}
+    let logo = "https://media.academictransfer.com/LT8OEP2nAexUPaM9-WfgcP488FM=/fit-in/490x162/filters:upscale():fill(white)/logos/ortec-en-wide.jpg";
 
-module.exports = { run, getJobs, getParams }; // this is needed for our unit test job
+    let postLogo = new scraper.ApiScraper(
+      "https://content.energage.com/company-images/61925/logo.png"
```

Comment:
Logo add endpoint is wrong
```suggestion
      "https://api.peviitor.ro/v1/logo/add/"
```
change it with this endpoint

Reply:
done, thx for the issue heads up